src/main/java/org/example/validators/JobRoleValidator.java

```diff
@@ -0,0 +1,107 @@
+package org.example.validators;
+
+import org.example.exceptions.Entity;
+import org.example.exceptions.InvalidException;
+import org.example.models.JobRoleRequest;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Date;
+
+public class JobRoleValidator {
+
+    private static final int JOB_ROLE_MAX_LENGTH = 50;
+    private static final int JOB_ROLE_MIN_LENGTH = 5;
+
+    public void validateJobRole(final JobRoleRequest jobRoleRequest) throws
+            InvalidException {
+
+        if (jobRoleRequest.getRoleName() == null
+                || jobRoleRequest.getRoleName().isEmpty()) {
+            throw new InvalidException(Entity.JOB_ROLE,
+                    "Job Role Name is required.");
+        }
+        if (jobRoleRequest.getRoleName().length() > JOB_ROLE_MAX_LENGTH) {
+            throw new InvalidException(Entity.JOB_ROLE,
+                    "Job Role Name is greater than 50 characters.");
```

Comment:
another very minor point... the error messages could use the JOB_ROLE_MAX_LENGTH and JOB_ROLE_MIN_LENGTH variables.... so if the values changed, the error message would automatically change